test fails: test_file_ascii and test_file_utf8: AssertionError

## Prerequisites
* [x] Did you make sure a similar [issue](../) didn't exist?
* [x] Did you update gTTS to the latest? (`pip install --upgrade gTTS`)

## Current Behaviour (steps to reproduce)
<!--- Tell us what happens instead of the expected behavior -->
<!--- Please use code blocks when pasting code or stack traces  -->
```
gtts/tests/test_cli.py::test_file_ascii FAILED                           [ 15%]
gtts/tests/test_cli.py::test_file_utf8 FAILED                            [ 16%]
(...)
=================================== FAILURES ===================================
_______________________________ test_file_ascii ________________________________

    @pytest.mark.net
    def test_file_ascii():
        with LogCapture() as lc:
            result = runner_debug(["--file", textfile_ascii])
            log = logcapture_str(lc)

>       assert "text: %s" % text in log
E       assert ('text: %s' % "Can you make pink a little more pinkish can you make pink a little more pinkish, nor can you make the font bigger?\nHow much will it cost the website doesn't have the theme i was going for.") in 'No logging captured'

gtts/tests/test_cli.py:233: AssertionError
________________________________ test_file_utf8 ________________________________

    @pytest.mark.net
    def test_file_utf8():
        with LogCapture() as lc:
            result = runner_debug(["--file", textfile_utf8])
            log = logcapture_str(lc)

>       assert "text: %s" % text_unicode in log
E       AssertionError: assert ('text: %s' % '这是一个三岁的小孩\n在讲述她从一系列照片里看到的东西。\n对这个世界， 她也许还有很多要学的东西，\n但在一个重要的任务上， 她已经是专家了：\n去理解
她所看到的东西。') in 'No logging captured'

gtts/tests/test_cli.py:243: AssertionError
======================== 2 failed, 95 passed in 33.54s =========================
```
## Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
platform freebsd13 -- Python 3.9.16, pytest-7.2.0, pluggy-1.0.0 -- /usr/local/bin/python3.9
cachedir: .pytest_cache
rootdir: /wrkdirs/usr/ports/audio/py-gtts/work-py39/gTTS-2.3.0, configfile: pyproject.toml
plugins: cov-2.9.0

* gTTS version: 2.3.0
* Operating System version: FreeBSD 13.1 amd64

Any clues?

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -63,11 +63,9 @@ documentation = "https://gtts.readthedocs.io"
 repository = "https://github.com/pndurette/gTTS"
 changelog = "https://github.com/pndurette/gTTS/blob/main/CHANGELOG.md"
 
-[tool.setuptools.packages.find]
-#where = ["src"]  # list of folders that contain the packages (["."] by default)
-#include = ["my_package*"]  # package names should match these glob patterns (["*"] by default)
-#exclude = ["my_package.tests*"]  # exclude packages matching these glob patterns (empty by default)
-namespaces = false  # to disable scanning PEP 420 namespaces (true by default)
+[tool.setuptools.package-data]
+# Tests support files
+"gtts.tests.input_files" = ["*.txt"]
 
 [tool.coverage.run]
 omit = [

